subprojects/gst-plugins-bad/sys/msdk/gstmsdkenc.c

```diff
@@ -1770,12 +1739,21 @@ gst_msdkenc_get_surface_from_frame (GstMsdkEnc * thiz,
 #endif
 
   inbuf = frame->input_buffer;
+
```

Comment:
It is not needed.

Reply:
It is in the original code, and I didn't change it. It is possible that frame->input_buffer is used many times in the function, using inbuf can simplify the code.

Reply:
You added a new line here which is not needed.

Reply:
Removed the empty line.